README.md

```diff
@@ -24,3 +24,106 @@ The code in this repository can be built and tested using the Makefile.
 For more information, see `make help`.
 
 [contrib]: https://github.com/projectcalico/calico/blob/master/CONTRIBUTING_CODE.md
+
+## How can I run Calico Node tests?
+
+Calico Node uses Python and [Nose](https://nose.readthedocs.io/en/latest/) for running tests.
```

Comment:
nit on the readme: "Calico Node" should be `calico/node` when referring to the image, and simply "node" (lowercase) when referring to the repository.

Reply:
Also note that we use golang / ginkgo for some of the tests (fv) - it's just the full integrations tests in python (which is most of them)